.github/workflows/pr-cpu.yaml

```diff
@@ -13,36 +13,26 @@ jobs:
     strategy:
       matrix:
         include:
-        - name: "cpu-3.10-1.13"
-          container: mosaicml/pytorch:1.13.1_cpu-python3.10-ubuntu20.04
-          markers: "not daily and not remote and not gpu and not vision and not doctest"
-          pytest_command: "coverage run -m pytest"
-          composer_package_name: "mosaicml"
-        - name: "cpu-3.10-2.0"
+        - name: cpu-3.10-2.0
           container: mosaicml/pytorch:2.0.1_cpu-python3.10-ubuntu20.04
-          markers: "not daily and not remote and not gpu and not vision and not doctest"
-          pytest_command: "coverage run -m pytest"
-          composer_package_name: "mosaicml"
-        - name: "cpu-3.10-2.1"
+          markers: not daily and not remote and not gpu and not vision and not doctest
+          pytest_command: coverage run -m pytest
+          composer_package_name: mosaicml
+        - name: cpu-3.10-2.1
           container: mosaicml/pytorch:2.1.0_cpu-python3.10-ubuntu20.04
-          markers: "not daily and not remote and not gpu and not vision and not doctest"
-          pytest_command: "coverage run -m pytest"
-          composer_package_name: "mosaicml"
-        # - name: 'cpu-3.10-2.2'
```

Comment:
2.2 should be active now?

Reply:
Chuck will update in a separate PR where he bumps images